src/Chapter3.hs

```diff
@@ -970,6 +1058,21 @@ implement the following functions:
 
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
+data WeekDay = Sunday|Monday|Tuesday|Wednesday|Thursday|Friday|Saturday deriving (Show,Read,Eq,Ord,Enum)
```

Comment:
For readability reasons, it's a good idea to put spaces around `|` (and all operators generally) ⬅️ 👾 ➡️ 

Though, for sum types, I usually put each constructor on a separate line 🙂